WG.md

```diff
@@ -8,12 +8,43 @@ for review by the TSC at an agreed upon date. A WG will be automatically disband
 
 ## Current Working Groups
 
+### Generics and circuit families
+
+**Objective**: ...
+
+Chair: Niel de Beaudrap  
+Members: Colm Ryan, Andrew Cross, Ali Javadi, Blake Johnson
+
+### Grammar
+
+**Objective**: 
```

Comment:
I suggest that the Objective of this group to be "Improve ANTLR reference implementation of OpenQASM grammar".

Reply:
https://github.com/Qiskit/openqasm/pull/271